persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_50_gsheets/generated/script.gs

```diff
@@ -0,0 +1,48 @@
+// AUTO-GENERATED FILE.
+
+// This file is an auto-generated file by Ballerina persistence layer for model.
+// Please verify the generated scripts and execute them against the target DB server.
+
+function createSheets() {
+	var activeSpreadsheet = SpreadsheetApp.getActiveSpreadsheet();
+	var yourNewSheet = activeSpreadsheet.getSheetByName("Employee");
+	if (yourNewSheet != null) {
```

Comment:
is this formatted correctly

Reply:
Yes. This is a single tab and I think due to having a `.gs` extension, we are getting this issue in visualising